tests/requirements.in

```diff
@@ -1,5 +1,4 @@
 docker
 flake8
-isort
```

Comment:
the file tests/requirements.txt is missing from this PR. You might want to regenerate it.